mox_stsorgsync/os2mo.py

```diff
@@ -70,15 +70,58 @@ def user_uuids():
     ]
 
 
+def chose_visible_prioritized_address(candidates, prioritized_classes):
+    chosen = None
+    # find candidate using prioritized list if available
+    for cls in prioritized_classes:
+        if chosen:
+            break
+        for candidate in candidates:
+            if (
+                candidate["address_type"]["uuid"] == cls
+                and candidate.get("visibility",
+                                  {"scope": "PUBLIC"})["scope"] == "PUBLIC"
+            ):
+                chosen = {"Value": candidate["name"],
+                          "Uuid": candidate["uuid"]}
+
+    if not prioritized_classes and len(candidates):
+        for candidate in reversed(candidates):
```

Comment:
Why do we reverse the list of candidates here?

Reply:
Previous behaviour was to overwrite until no more addresses came. In order to keep that behaviour I reversed the list.